### PR TITLE
Supporting custom path-host along with path-regex

### PR DESCRIPTION
**Pull request by tosan88**
Example:
```etcdctl set   /ft/services/service-a/path-regex/bar   /bar/.* ```
```etcdctl set   /ft/services/service-a/path-host/bar  public-host```
will result in
```
/vulcand/frontends/vcb-service-a-path-regex-bar/frontend  {"Type":"http", "BackendId":"vcb-service-a", "Route":"PathRegexp(`/bar/.*`) && Host(`public-host`)", "Settings": {"FailoverPredicate":""}}
```

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -81,6 +81,7 @@ type Service struct {
 	HasHealthCheck    bool
 	Addresses         map[string]string
 	PathPrefixes      map[string]string
+	CustomHosts       map[string]string
```
Comment (catalin-ilea) [inline main.go]: PathHosts maybe?

Comment (tosan88) [inline main.go]: Correct. Done!

**Comment by catalin-ilea** [comment]
lgtm

**Comment by BerniVarga** [comment]
+1

**Pull request merged**